src/bci_build/package/gcc/README.md.j2

```diff
@@ -0,0 +1,95 @@
+# The GNU Compiler Collection Container Image
+
+The GNU Compiler Collection (GCC) is an optimizing compiler for various
+architectures and operating systems. It is the default compiler in the GNU
+project and most Linux distributions, including SUSE Linux Enterprise and
+openSUSE.
+
+
+## How to use the image
+
+### Compile an application with a `Dockerfile`
+
+Normally, you'd want to compile an application and distribute it as part of a
+custom container image. To do this, create a `Dockerfile` similar to the one
+below. The `Dockerfile` uses this image to build a custom container image,
+copies the sources to a working directory, and compiles the application:
+{%- set reference_no_release = (image.reference | replace("-%RELEASE%", "") ) %}
+
+```Dockerfile
+FROM {{ reference_no_release }}
+WORKDIR /src/
+COPY . /src/
+RUN gcc main.c src1.c src2.c
+CMD ["./a.out"]
+```
+
+It is also possible to compile a static binary with gcc as part of a multistage
+build:
+
+```Dockerfile
+FROM {{ reference_no_release }} as builder
+WORKDIR /src/
+COPY . /src/
+RUN gcc -static main.c src1.c src2.c
```

Comment:
I am not sure we actually need -static here or should advertise -static (it interacts badly with glibc). also an `-o app` should be listed

Reply:
the `-o app` shouldn't be necessary, as the `COPY` changes the filename